CMakeLists.txt

```diff
@@ -305,24 +310,26 @@ if(FLATBUFFERS_BUILD_SHAREDLIB)
                         VERSION "${FlatBuffers_Library_SONAME_FULL}")
 endif()
 
-function(compile_flatbuffers_schema_to_cpp_opt SRC_FBS OPT)
+function(compile_flatbuffers_schema_to_cpp_with_add_options SRC_FBS NO_INCLUDES GEN_COMPARE ADD_OPTIONS)
```

Comment:
why 3 arguments for options? There is no ordering to the flatc args, so just one would suffice.

Reply:
Yes, you are right